packages/terra-slide-panel/tests/wdio/slide-panel-spec.js

```diff
@@ -0,0 +1,151 @@
+/* global browser, Terra, before */
+
+const context = '[data-terra-dev-site-content] *:first-child';
+
+describe('Slide panel end', () => {
+  before(() => browser.setViewportSize(Terra.viewports('large')[0]));
```

Comment:
Since these tests all use viewport "large" we could move the setViewPortSize to an outer describe to improve performance.

Reply:
Good call: [updated](https://github.com/cerner/terra-framework/pull/131/commits/5eeaefacfa650156311b033690a5216dddf97f24)